docs/docs/using-fragments.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Using fragments
+---
+
+Fragments allow you to reuse parts of GraphQL queries. It also allows you to split up complex queries into smaller, easier to understand components.
+
+### The building blocks of a fragment
+
+Here is an example fragment:
+
+```graphql
+fragment FragmentName on TypeName {
+  field1
+  field2
+}
+```
+
+A fragment consists of three components:
+
+1. `FragmentName`: the name of the fragment that will be referenced later.
+2. `TypeName`: the [GraphQL type](https://graphql.org/graphql-js/object-types/) of the object the fragment will be used on. This is important because you can only query for fields that actually exist on a given object.
+3. The body of the query. You can define any fields with any level of nesting in here, the same that you would elsewhere in a GraphQL query
+
+### Creating and using a fragment
+
+A fragment can be created inside any GraphQL query, but it's good practice to create the query separately. More organization advice in the [Conceptual Guide](/docs/querying-with-graphql/#fragments).
+
+```jsx
```

Comment:
Please use the title syntax instead of the comment:

`jsx:title=path-to-file`

Reply:
I followed the style I saw in https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/querying-with-graphql.md#fragments - I'll submit a PR to swap that page over to the title syntax as well.

Reply:
PR #8912 submitted and merged